src/contracts/pods/EigenPodManager.sol

```diff
@@ -140,6 +140,7 @@ contract EigenPodManager is
                 });
             }
         }
+        emit PodBalanceUpdated(podOwner, sharesDelta);
```

Comment:
is this the right place to have this event?
trying to understand if this is all that's needed to fully track shares changes with indexing of events -- do we have a good doc on this? if so, where? if not, can you please make one?

Reply:
interesting, the event only emits deltas, so to get accurate balance users need to replay and stack all deltas?

is it possible to emit balance directly instead?

Reply:
token transfer events similar just log in/out amounts, but I hear you on this @bowenli86 
 this isn't the only place in the code where these share amounts change, so a single event here won't be sufficient regardless. we have other events elsewhere but I'd have to review to remind myself of the details.

Reply:
Shouldn't we be ok just emitting an event with "updatedPodOwnerShares"/"sharesDelta? Looks like there are events in the DM for changes in delegated shares: OperatorSharesIncreased, OperatorSharesDecreased

Reply:
I'm compiling a doc on the other events we need. There are some edge cases we are missing, particularly with withdrawals as shares (this is for both LST and Native ETH)